### PR TITLE
Port over the FAQ page

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Add the FAQ page

### CODE REVIEW

**Comment by CristinaRO** [comment]
[FAQ page](https://trello.com/c/uTDlS8rh/19-faq-page)

File: app/views/faq/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          different people, to help limit abuse, or the impact of people with
+          terrible taste.
+        </p>
+      </dd>
+
+      <dt>Who built it?</dt>
+      <dd>
+        <p>
+          This site was originally built by <a href="https://www.dxw.com/">The
+          Dextrous Web (now dxw)</a> for <a href="http://www.mysociety.org">mySociety</a>.
```
Comment (erbridge) [inline app/views/faq/show.html.erb]: I think we can probably just say `dxw` here instead of using the full name.

**Review by erbridge — APPROVED** [review]
Assuming the content has been reviewed or is a copy and paste.

File: app/views/faq/show.html.erb (hunk 1 above)
Comment (CristinaRO) [reply app/views/faq/show.html.erb]: Hah, yes, it was a copy and paste, except for where it said `"(now Dextrous Digital)"`, which I've already changed to `"(now dxw)"`. I can change it to just "built by dxw".

Comment (erbridge) [reply app/views/faq/show.html.erb]: That's my reckon.

Comment (CristinaRO) [reply app/views/faq/show.html.erb]: Done. I've also added the link to the votes TSV, which they've uploaded in the meantime.

**Pull request merged**